asset-transfer-ledger-queries/application-javascript/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "asset-transfer-basic",
+    "version": "1.0.0",
+    "description": "Asset-transfer-basic application implemented in JavaScript",
+    "engines": {
+        "node": ">=12",
+        "npm": ">=5"
+    },
+    "engineStrict": true,
+    "author": "Hyperledger",
+    "license": "Apache-2.0",
+    "dependencies": {
+        "fabric-ca-client": "2.2.0",
```

Comment:
I think you want "^2.2.0" for these, so that it works with later SDK versions as well.

Reply:
done